### ISSUE TITLE
failure to escape on names with / in them

### ISSUE THREAD

**Issue by tgbugs** [NeuroscienceKnowledgeSpace/KnowledgeSpace#17]
https://nye.crbs.ucsd.edu/NeuroKS/index.php/Results/view/SCR_014306-1/Neocortex%20Layer%20II%2FIII%20chandelier%20cell/1

404ers, it shouldnt.

**Comment by wawong** [comment]
This problem is related to the apache configuration. I looked it up and I am able to solve this problem by adding "AllowEncodedSlashes On or NoDecode" in the apache config file on my local machine.

(See http://stackoverflow.com/questions/9206835/2f-in-url-breaks-and-does-not-reference-to-the-php-file-required)
But it does not work on nye.crbs.ucsd.edu. I created a ticket in JIRA, so that the CRBS system admin can take a look.
https://support.crbs.ucsd.edu/browse/NIF-15505
